test/compile/scale/init.test.ts

```diff
@@ -35,5 +35,13 @@ describe('compile/scale', () => {
       assert.equal(scale.paddingOuter, 0.15);
       assert.isUndefined(scale.padding);
     });
+
+    it('should have scale type as utc if timeunit is utc', () => {
+      const scale = initScale(
+        'x', {field: 'a', type: 'temporal', scale: {type: 'band'}, timeUnit: 'utcyear'}, {scale: {bandPaddingInner: 0.3}},
```

Comment:
We need to make it utc because the type is temporal and the timeUnit is utcyear, so scale type should be consistent with it